### DISCUSSION TITLE
Modal组件是挂载到body上的，为什么不显示呢，会跟随到父组件的状态变化？

### DISCUSSION THREAD

**Discussion by Justin3go** [lobehub/lobehub#1031] [Development Question] [answered]
搞不明白了，求助一下

## 背景

在做助手列表分组管理时，我在`Dropdown`组件同级目录中放置了一个`Modal`组件+`Input`用来设置分组名

## 问题

预期在点击`Dropdown`中某个菜单时显示该`Modal`，但实际情况并未显示。并且，我将Modal组件的`open`设置为`true`显示，出现的结果是我鼠标hover在助手列表目录中弹窗才能出现，Modal不是挂载在body上吗，但似乎这依赖于其父组件，我有点不是很理解：

[完整可复现链接](https://github.com/lobehub/lobe-chat/compare/main...Justin3go:lobe-chat:feat_group)

## 视频演示

https://github.com/lobehub/lobe-chat/assets/63507251/75c24024-463f-437d-a64c-be5e7ba9e142

## 其他尝试方案

我试图通过`Modal.confirm`这个全局方法调用，使用其`content`属性包含`Input`组件，但是这也会遇到问题就是：如何传递Input中的值到onOk函数里面

**Comment by canisminor1990** [comment, marked as answer]
不是挂载的问题，而是你把弹窗组件放在菜单里的话，当菜单销毁时弹窗也被销毁了，移到  Dropdown 外面 @Justin3go
